Documentation/contributing/development/dev_setup.rst

```diff
@@ -17,19 +17,28 @@ suitable method.
 Quick Start
 -----------
 
-If you're in a hurry, here are the essential steps to get started:
+The below commands install Cilium within a `Kind`_-based Kubernetes cluster.
+Invoke them within the root of the Cilium repository. Their output will inform
+you of any missing dependencies (note that "Cilium not being found" refers to
+the Cilium CLI). The make targets are described in later sections.
```

Comment:
"Execute" is standard, but you can also use "run", which I slightly prefer.
Specific is better than general.
Use clear, plain language.
Use present tense.
Wrap commands in code font.
What's the actual message that users will see for "Cilium not being found"? Better to quote that and separate the "Note" sentence into a `.. note::` callout.
Which later sections? List them, preferably with linked text.
```suggestion
Run them in the root directory of the Cilium repository. The command output informs
you of any missing dependencies. (Note that "Cilium not being found" refers to
the Cilium CLI.) The `make` targets are described in later sections.
```